### ISSUE TITLE
Multiple Validation Errors and Unauthorized Access When Applying Template

### ISSUE THREAD

**Issue by terrxo** [directus-labs/directus-template-cli#57]
## Description

I encountered multiple issues while applying a Directus template using the `directus-template-cli`. The process produced numerous validation errors and unauthorized access warnings. Below are the details of the encountered issues.

## Steps to Reproduce

1. Run the `directus-template-cli` command:
   ```sh
   npx directus-template-cli@latest apply
   ```
2. Choose to apply a template from a local directory.
3. Provide the local template directory: `templates/headless-cms`.
4. Enter the Directus URL and Admin Token when prompted.

## Observed Behavior

1. **Unique Constraint Violations**: When loading users, the following warning was generated:
   ```
   Warning: Status 400 • "Value for field \"email\" in collection \"directus_users\" has to be unique."
   ```

2. **Validation Errors**: Multiple fields required by the template were missing values, causing numerous validation errors:
   ```
   Warning: Status 400 • "Validation failed for field \"image\". Value is required."
   Warning: Status 400 • "Validation failed for field \"primary_image\". Value is required."
   Warning: Status 400 • "Validation failed for field \"headline\". Value is required."
   Warning: Status 400 • "Validation failed for field \"cta\". Value is required."
   Warning: Status 400 • "Validation failed for field \"name\". Value is required."
   Warning: Status 400 • "Validation failed for field \"type\". Value is required."
   Warning: Status 400 • "Validation failed for field \"schema\". Value is required."
   Warning: Status 400 • "Validation failed for field \"data\". Value is required."
   Warning: Status 400 • "Validation failed for field \"title\". Value is required."
   Warning: Status 400 • "Validation failed for field \"thumbnail\". Value is required."
   Warning: Status 400 • "Validation failed for field \"value\". Value is required."
   Warning: Status 400 • "Validation failed for field \"label\". Value is required."
   ```

3. **Invalid Foreign Key Errors**: There were numerous warnings about invalid foreign key values:
   ```
   Warning: Status 400 • "Invalid foreign key for field \"cta\" in collection \"block_banner_cta\"."
   Warning: Status 400 • "Invalid foreign key for field \"element_brand_id\" in collection \"block_brands_detailed_carousel_element_brand\"."
   Warning: Status 400 • "Invalid foreign key for field \"block_generic_heading_id\" in collection \"block_generic_heading_translations\"."
   Warning: Status 400 • "Invalid foreign key for field \"element_image_id\" in collection \"block_hero_carousel_element_image\"."
   ```

4. **Permission Issues**: There were warnings about lacking permission to access certain fields or collections:
   ```
   Warning: Status 403 • "You don't have permission to access this."
   ```
## Expected Behavior

The template should apply successfully without any validation errors, foreign key issues, or unauthorized access warnings.

## Environment

- **Directus Version**: 10.11.2
- **Directus Template CLI Version**: 0.4.3
- **Operating System**: macOS
- **Node.js Version**: 18.19.1
- **Database**: Postgis 16-3.4

## Logs and Screenshots

Please find attached the complete terminal log for further reference.
[template](https://github.com/user-attachments/files/15580573/template.tar.gz)
[terminal_logs](https://github.com/user-attachments/files/15580577/terminal_logs.txt)

## Additional Information

Any help in resolving these issues would be greatly appreciated. Thank you!

**Comment by arladmin** [comment]
I am facing similar issues, particularly since Directus v11.x

**Comment by bryantgillespie** [comment]
This is probably could be a couple things
- The error about emails is probably because the default admin@example user that gets created when you spin up an instance.
- If you're sure you have the correct access token with admin permissions, then it's likely an issue where you've set Not Allow Null to true for some of the fields
  - The way we load the data is creating "skeleton" records (essentially empty) first to make maintaining relationships much easier and then we go back and update all the data. So if you have fields that DO NOT allow null values then the Directus API will throw an exception when trying to create that item.

> I am facing similar issues, particularly since Directus v11.x
We don't have v11 support yet in the CLI. 
Tracking  V11 support here #61

**Comment by bryantgillespie** [comment]
Hopefully these issues should be fixed in v0.5.0
See #63 

A beta version is now published and you can test it using @beta tag
`npx directus-template-cli@beta apply`
or
`npx directus-template-cli@beta extract`
